<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplicationWindow</class>
 <widget class="QDialog" name="ReplicationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create replication</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="5,0,0,0">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Replicate changes from</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QWidget" name="verticalWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="leftMargin">
         <number>15</number>
        </property>
        <item row="0" column="0">
         <widget class="QRadioButton" name="rdb_localsource">
          <property name="text">
           <string>Local database:</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="rdb_remotesource">
          <property name="text">
           <string>Remote database:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="txt_remotesource">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>http://</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cmb_localsource"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Replicate changes to</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout_2" columnstretch="4,96">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QCheckBox" name="cbx_persistent">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Persistent</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <property name="leftMargin">
        <number>24</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="txt_name">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>User_ctx:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="txt_userctx">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>{&quot;roles&quot;:[&quot;_admin&quot;]}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="cbx_continuous">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Continuous</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="3">
    <widget class="QPushButton" name="btn_cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="btn_save">
     <property name="text">
      <string>Add</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QWidget" name="verticalWidget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="leftMargin">
         <number>15</number>
        </property>
        <item row="0" column="0">
         <widget class="QRadioButton" name="rdb_localtarget">
          <property name="text">
           <string>Local database:</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="rdb_remotetarget">
          <property name="text">
           <string>Remote database:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cmb_localtarget"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="txt_remotetarget">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>http://</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="cbx_create_target">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Create target</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_5">
       <property name="leftMargin">
        <number>24</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Database:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txt_createtarget">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cbx_persistent</sender>
   <signal>toggled(bool)</signal>
   <receiver>txt_name</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbx_persistent</sender>
   <signal>toggled(bool)</signal>
   <receiver>txt_userctx</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdb_remotesource</sender>
   <signal>toggled(bool)</signal>
   <receiver>txt_remotesource</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdb_localsource</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmb_localsource</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdb_localtarget</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmb_localtarget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdb_remotetarget</sender>
   <signal>toggled(bool)</signal>
   <receiver>txt_remotetarget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>ReplicationWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>791</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbx_persistent</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbx_continuous</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbx_create_target</sender>
   <signal>toggled(bool)</signal>
   <receiver>verticalWidget_2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbx_persistent</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbx_create_target</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbx_create_target</sender>
   <signal>toggled(bool)</signal>
   <receiver>txt_createtarget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbx_persistent</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_5</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
